src/iTin.Core/Extensions/Int32Extensions.cs

```diff
@@ -93,7 +93,6 @@ public static int GetBit(this int value, byte bit)
         public static byte GetByte(this int value, Bytes onebyte)
         {
             SentinelHelper.IsEnumValid(onebyte);
-            SentinelHelper.IsTrue((byte)onebyte > 1);
```

Comment:
no need to verify here since the next function stack is validating the boundary already.

Reply:
Thanks!!, done!